src/settings/style.scss

```diff
@@ -23,3 +23,7 @@
 		}
 	}
 }
+
+.components-form-token-field__input-container {
```

Comment:
This is a general override, can we get this in `.gp-settings` so it only applies to stuff we put in.?

Reply:
I thought that's where I put it